apps/src/studio/studio.js

```diff
@@ -1851,6 +1851,14 @@ Studio.checkForPreExecutionFailure = function () {
     return true;
   }
 
+  if (studioApp.hasEmptyFunctionOrVariableName()) {
+    Studio.result = false;
+    Studio.testResults = TestResults.EMPTY_FUNCTION_NAME;
+    Studio.message = commonMsg.unnamedFunction();
+    Studio.preExecutionFailure = true;
+    return true;
+  }
```

Comment:
It's kind of a bummer that this has to be re-implemented for each game type.  Do we have games were an unnamed function is okay, or is there just not a common place to put these?